<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesWidget</class>
 <widget class="QWidget" name="PropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>143</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBoxBit1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Пониженное энерго
потребление</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>158</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аккумулятор</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>101</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Противоударный
корпус</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Локальная сборка</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>132</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>184</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBit3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Влагозащита</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
